site/pages/technology/style.css

```diff
@@ -94,6 +94,11 @@
   height: 20px;
 }
 
+.arrow svg g {
```

Comment:
Can we apply this colour to the style__latestRoundTableLink___2k8YE as well please?